hummingbird/ml/operator_converters/_scaler_implementations.py

```diff
@@ -0,0 +1,42 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+Base classes for scaler implementations.
+"""
+
+import torch
+
+from ._base_operator import BaseOperator
+
+
+class Scaler(BaseOperator, torch.nn.Module):
+    """
+    Class implementing Scaler operators in PyTorch. Supported normalizers are L1, L2 and Max.
+    """
+
+    def __init__(self, offset, scale, device):
+        super(Scaler, self).__init__()
+        self.transformer = True
+
```

Comment:
Can you do like self.offset = self.scale = None and then just have one branch each?